scico/blockarray.py

```diff
@@ -1314,8 +1321,8 @@ def _index_wrapper(self, func_str, values):
         index = self.index
         arr_tuple = self._block_array.split
         if bk_index == Ellipsis:
-            # TODO does this result in multiple copies? One per sub-blockarray,
-            # then one to combine into a nested BA?
+            # This may result in multiple copies: one per sub-blockarray,
```

Comment:
This behaviour is confirmed? If not, it would be good to open an issue as a reminder to check.

Reply:
Issue added #172. I do not fully understand the question nor the code, so it might help if you edited the issue.

Reply:
I don't understand the question either. I added a request for comment from @lukepfister to the issue. I think this PR is fine we for now. We can come back to this comment later if the outcome of the discussion in #172 warrants it.